src/components/common/MwContextualMenu/MwContextualMenu.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../styles/colors.scss';
+
+.mwContextualMenu {
+  position: absolute;
+  z-index: 11;
+  background: rgba(35, 35, 35, 0.918051);
+  box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.1), 0px 2px 10px rgba(0, 0, 0, 0.1), 0px 10px 20px rgba(0, 0, 0, 0.1),
+    0px 10px 50px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(13.5914px);
+  border-radius: 4px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 20px;
+  color: $white-color;
+  mix-blend-mode: normal;
+}
+
+.mwContextMenuOption {
+  padding: 4px 20px 4px 20px;
+  opacity: 0.8;
+  &:hover {
+    background: $red-color;
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.1), 0px 10px 20px rgba(0, 0, 0, 0.1), 0px 10px 50px rgba(0, 0, 0, 0.1);
+    backdrop-filter: blur(13.5914px);
```

Comment:
This value is repeated a lot, consider moving it to a variable, please.

Reply:
they are a little bit different, but I agree that it is better to extract some to variables